sentry_sdk/integrations/opentelemetry/potel_span_processor.py

```diff
@@ -22,15 +34,23 @@ def __new__(cls):
 
     def __init__(self):
         # type: () -> None
-        pass
+        self._children_spans = {}  # type: dict[int, List[ReadableSpan]]
```

Comment:
Could we do `defaultdict(list)` here and then just append at l50? Personal preference but I find the api much more intuitive than `setdefault`